ui/packages/catalog-ui-search/src/main/webapp/component/metacard-associations/metacard-associations.view.js

```diff
@@ -34,7 +35,49 @@ module.exports = Marionette.LayoutView.extend({
     associationsList: '> .editor-content',
     associationsGraph: '> .content-graph',
   },
-  template,
+  template() {
+    const canEdit = user.canWrite(this.model.get('metacard').get('properties'))
+
+    return (
+      <React.Fragment>
+        <div className="content-menu" />
+        <div className="content-graph" />
+        <div className="list-header">
+          <div className="header-text header-parent">Parent</div>
+          <div className="header-text header-relationship">Relationship</div>
+          <div className="header-text header-child">Child</div>
+        </div>
+        <div className="editor-content" />
+        {canEdit ? (
+          <React.Fragment>
+            <div className="list-footer">
+              <div className="footer-text" />
+              <button className="footer-add is-positive">
+                <span className="fa fa-plus" />
+                <span>&nbsp;Add Association</span>
+              </button>
+            </div>
+            <div className="editor-footer">
+              <button className="footer-edit is-primary">
+                <span className="fa fa-pencil" />
+                <span>&nbsp;Edit</span>
+              </button>
+              <button className="footer-cancel is-negative">
+                <span className="fa fa-times" />
+                <span>&nbsp;Cancel</span>
+              </button>
+              <button className="footer-save is-positive">
+                <span className="fa fa-floppy-o" />
+                <span>&nbsp;Save</span>
+              </button>
+            </div>
+          </React.Fragment>
+        ) : (
+          <React.Fragment />
```

Comment:
Is there a reason this can't just be null?

Reply:
I was thinking the same thing